README.md

```diff
@@ -46,6 +46,12 @@ If you want to use a package manager:
 - [Snap](https://snapcraft.io/) users can use `sudo snap install helm
   --classic`.
 
+If you want to install helm from script:
+
+- [Script]
+(https://helm.sh/docs/using_helm/#from-script)
```

Comment:
Need to have [Script](https://helm.sh/docs/using_helm/#from-script) on same line beside each other without any space for the link to work as expected.